cli/cmd/io/read.hpp

```diff
@@ -38,14 +39,19 @@ class read_command : public command
             command(e, "Reads a file or a directory of files and creates logic network or FCN layout objects "
                        "which will be put into the respective store. Current supported file types are:\n"
                        "Logic networks: Verilog, AIGER, BLIF.\n"
+                       "Gate-level layouts: FGL.\n"
                        "Cell-level layouts: FQCA.\n"
                        "In a directory, only files with extension '.v', '.aig', '.blif' are considered.")
     {
         add_option("filename", filename, "Filename or directory")->required();
+        add_option("topology", topology,
+                   "Topology for gate-level layouts. Can be 'cartesian' or of the form "
+                   "'<odd|even>_<row|column>_<cartesian|hex>");
```

Comment:
```suggestion
                   "'<odd|even>_<row|column>_<cartesian|hex>'");
```
There seems to be a closing single quote missing here.